modin/core/execution/modin_aqp.py

```diff
@@ -11,14 +11,16 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""The module for working with displaying progress bars for Ray engine."""
+"""The module for working with displaying progress bars for Modin execution engines."""
```

Comment:
@modin-project/modin-core, does anyone remember why the file has the suffix `aqp`?

@noah-kuo, tagging you as the original author.

Reply:
I would guess it's something like `automatic query progress` or smth similar

Reply:
Added a clarification.